lib/src/SofaGLFW/SofaGLFWBaseGUI.cpp

```diff
@@ -188,6 +188,46 @@ void SofaGLFWBaseGUI::destroyWindow()
 {
 }
 
+GLFWmonitor* SofaGLFWBaseGUI::getCurrentMonitor(GLFWwindow *glfwWindow)
+{
+    auto mini = [](int x, int y){ return x < y ? x : y; };
+    auto maxi = [](int x, int y){ return x > y ? x : y; };
+
+    int num_monitors, i;
```

Comment:
See https://www.sofa-framework.org/community/doc/programming-with-sofa/guidelines/ to learn the coding style used in SOFA. For variable names, we use lower Came case.